client/rest-high-level/src/test/java/org/elasticsearch/client/IndicesClientIT.java

```diff
@@ -180,6 +180,32 @@ public void testOpenNonExistentIndex() throws IOException {
         assertEquals(RestStatus.NOT_FOUND, strictException.status());
     }
 
+    public void testRefresh() throws IOException {
+        {
+            String[] indices = randomIndices(1, 5);
+            for (String index : indices) {
+                createIndex(index);
+            }
+            RefreshRequest refreshRequest = new RefreshRequest(indices);
+            RefreshResponse refreshResponse =
+                execute(refreshRequest, highLevelClient().indices()::refresh, highLevelClient().indices()::refreshAsync);
+            // 10 shards per index by default
+            assertThat(refreshResponse.getTotalShards(), equalTo(indices.length * 10));
```

Comment:
shall we also check successful and failed?

Reply:
Seems `RefreshResponse` is a `BroadcastResponse` instead of an `AcknowledgedResponse`, so don't need to check if it's acknowledged ?

Reply:
sure I was asking if we can check successful and failed rather than only total. does that make sense?

Reply:
thanks for the explanation, your proposal sounds good to me

Reply:
Sounds good too